docs: `--singleton-table` DDL strategy flag (#1881)
vitessio/website: content/en/docs/22.0/user-guides/schema-changes/ddl-strategy-flags.md

  It does not make sense to combine `--singleton` and `--singleton-context`.

> - `--singleton-table`: reject submission for a table or view that is already pending migration.

Reviewer: ```suggestion
- `--singleton-table`: reject a new submission for a table or view which already has a running migration.
```

Author: Accepting - but then re-editing to read "pending", which is the correct term (that is also explained in this page).
